Nasal/instruments.nas

```diff
@@ -1585,26 +1598,25 @@ if( arg[0] == 1 ) # co-pilot
 
 # initialize KURS-MP frequencies & headings
 var kursmp_init = func{
```

Comment:
Please do not mix in whitespace changes - it's hard to tell if the lines below just shifted to the right or actually changed.  Whitespace changes should always come as a separate patch (if at all, because it will be broken anyway :)).